addons/event/views/event_views.xml

```diff
@@ -495,7 +495,7 @@
             <field name="name">event.registration.tree</field>
             <field name="model">event.registration</field>
             <field name="arch" type="xml">
-                <tree string="Registration" multi_edit="1" sample="1">
+                <tree string="Registration" multi_edit="1" sample="1" class="o_event_attendee_view_tree">
```

Comment:
Would name it o_event_registration_view_tree, to be model-based (as attendee is the "business name").